firestore/android/FirestoreSnippetsCpp/app/src/main/cpp/snippets.cpp

```diff
@@ -855,6 +855,57 @@ void ReadDataQueryOperators(firebase::firestore::Firestore* db) {
   cities_ref.WhereGreaterThanOrEqualTo("name",
                                        FieldValue::String("San Francisco"));
   // [END example_filters]
+
+}
+
+// https://firebase.google.com/docs/firestore/query-data/queries#array_membership
+void ReadDataArrayMembershipOperators(firebase::firestore::Firestore* db) {
+  using firebase::firestore::CollectionReference;
+  using firebase::firestore::FieldValue;
+
+  // Some example filters:
+  // [START cpp_array_contains_filter]
+  CollectionReference cities_ref = db->Collection("cities");
+
+  cities_ref.WhereArrayContains("region", FieldValue::String("west_coast"));
+  // [END cpp_array_contains_filter]
+
+}
+
+// https://firebase.google.com/docs/firestore/query-data/queries#in_not-in_and_array-contains-any
+void ReadDataArrayInNotInOperators(firebase::firestore::Firestore* db) {
+  using firebase::firestore::CollectionReference;
+  using firebase::firestore::FieldValue;
+
+  // Some example filters:
+  // [START cpp_in_filter]
+  CollectionReference cities_ref = db->Collection("cities");
+
+  cities_ref.WhereIn("country", std::vector<typename firebase::firestore::FieldValue> {FieldValue::String("USA"), FieldValue::String("Japan")});
```

Comment:
Can you omit the "firebase::firestore::" prefix here, since there is the "using" clause above? Applies elsewhere in this file as well.

Reply:
Done.